packages/twenty-front/src/pages/settings/profile/appearance/components/DateTimeSettingsTimeZoneSelect.tsx

```diff
@@ -25,7 +26,9 @@ export const DateTimeSettingsTimeZoneSelect = ({
       value={value}
       options={[
         {
-          label: `System settings - ${systemTimeZoneOption.label}`,
+          label: isDefined(systemTimeZoneOption)
```

Comment:
@lucasbordeau do you have knowledge on this ? I don't know when systemTimeZoneOption can be undefined

Reply:
The typing of findAvailableTimeZoneOption should certainly be `SelectOption<string> | undefined` since it indexes a dictionary and the key could be nonexistent.

Reply:
We should open an issue, as : 
- The typing should be fixed
- We should investigate why it is trying to index a non existent key